Tools/scripts/build_options.py

```diff
@@ -138,6 +137,16 @@ def __init__(self,
     Feature('GPS Drivers', 'SBF', 'AP_GPS_SBF_ENABLED', 'Enable SBF GPS', 0, None),
     Feature('GPS Drivers', 'SIRF', 'AP_GPS_SIRF_ENABLED', 'Enable SiRF GPS', 0, None),
 
+    Feature('Airspeed Drivers', 'analog', 'AP_GPS_ANALOG_ENABLED', 'Enable analog Airspeed', 0, ['AP_AIRSPEED_ENABLED']),
```

Comment:
why do we have "GPS" in airspeed defines?

Reply:
... because copy/paste/adjust can sometimes go very, very wrong :-)

Fixed.